market_maker_keeper/etoro_market_maker_keeper.py

```diff
@@ -0,0 +1,195 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2020 MakerDAO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import logging
+import sys
+from retry import retry
+from datetime import datetime, timezone
+
+from market_maker_keeper.band import Bands
+from market_maker_keeper.control_feed import create_control_feed
+from market_maker_keeper.limit import History
+from market_maker_keeper.order_book import OrderBookManager
+from market_maker_keeper.order_history_reporter import create_order_history_reporter
+from market_maker_keeper.price_feed import PriceFeedFactory
+from market_maker_keeper.reloadable_config import ReloadableConfig
+from market_maker_keeper.spread_feed import create_spread_feed
+from market_maker_keeper.util import setup_logging
+from pyexchange.etoro import EToroApi, Order
+from pymaker.lifecycle import Lifecycle
+from pymaker.numeric import Wad
+
+
+class EToroMarketMakerKeeper:
+    """Keeper acting as a market maker on eToro."""
+
+    logger = logging.getLogger()
+
+    def __init__(self, args: list):
+        parser = argparse.ArgumentParser(prog='eToro-market-maker-keeper')
+
+        parser.add_argument("--etoro-api-server", type=str, default="https://7vriaeqd.hft.etorox.com",
```

Comment:
Skeptical that this URL should be in a public repo...how are these API URLs generated?

Reply:
Removed as suggested